### PR TITLE
Close changelog for 7.3.0

### PR DESCRIPTION
**Pull request by mikemadden42**

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -3,6 +3,182 @@
...
+- New `extract_array` processor. {pull}11761[11761]
+- Add number of goroutines to reported metrics. {pull}12135[12135]
+- Add `proxy_disable` output flag to explicitly ignore proxy environment variables. {issue}11713[11713] {pull}12243[12243]
+- Processor `add_cloud_metadata` adds fields `cloud.account.id` and `cloud.image.id` for AWS EC2. {pull}12307[12307]
+- Add `decode_base64_field` processor for decoding base64 field. {pull}11914[11914]
+- Add aws overview dashboard. {issue}11007[11007] {pull}12175[12175]
+- Add `decompress_gzip_field` processor. {pull}12733[12733]
+- Add `timestamp` processor for parsing time fields. {pull}12699[12699]
+- Add Oracle Tablespaces Dashboard {pull}12736[12736]
+- Add `proxy_disable` output flag to explicitly ignore proxy environment variables. {issue}11713[11713] {pull}12243[12243]
```
Comment (acchen97) [inline CHANGELOG.asciidoc]: dupe of line 113

File: CHANGELOG-developer.asciidoc [hunk 2]
```diff
@@ -12,6 +12,14 @@ other Beats should be migrated.
 
 Note: This changelog was only started after the 6.3 release.
 
+=== Beats version 7.3.0
+https://github.com/elastic/beats/compare/v7.3.0..v7.2.0[Check the HEAD diff]
```
Comment (urso) [inline CHANGELOG-developer.asciidoc]: versions should be the other way around: v7.2.1...v7.3.0 :)

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -3,6 +3,388 @@
...
+- Export automation templates used to create functions. {pull}11923[11923]
+- Configurable Amazon endpoint. {pull}12369[12369]
+
+==== Deprecated
+
+*Filebeat*
+
+- `postgresql.log.timestamp` field is deprecated in favour of `@timestamp`. {pull}12338[12338]
+
+=== Beats version 7.2.1
```
Comment (urso) [inline CHANGELOG.asciidoc]: missing anchor `[[release-notes-7.2.1]]`

File: libbeat/docs/release.asciidoc [hunk 4]
```diff
@@ -18,6 +21,7 @@ upgrade.
 * <<release-notes-7.0.0-beta1>>
 * <<release-notes-7.0.0-alpha2>>
 * <<release-notes-7.0.0-alpha1>>
+* <<release-notes-6.8.2>>
```
Comment (urso) [inline libbeat/docs/release.asciidoc]: missing 6.8.1?

**Pull request merged**